docs/getting-started-guides/clc.md

```diff
@@ -0,0 +1,317 @@
+Cloud control portal and delete the parent server group that contains the
+Kubernetes Cluster. We hope to add a scripted option to do this soon.
+
+## Examples
+Create a cluster with name of k8s_1, 1 master node and 3 worker minions (on physical machines), in VA1
+
+```
+ bash kube-up.sh --clc_cluster_name=k8s_1 --minion_type=bareMetal --minion_count=3 --datacenter=VA1
+```
+
+Create a cluster with name of k8s_2, an ha etcd cluster on 3 VMs and 6 worker minions (on VMs), in VA1
+
+```
+ bash kube-up.sh --clc_cluster_name=k8s_2 --minion_type=standard --minion_count=6 --datacenter=VA1 --etcd_separate_cluster=yes
+```
+
+Create a cluster with name of k8s_3, 1 master node, and 10 worker minions (on VMs) with higher mem/cpu, in UC1:
+
+```
+  bash kube-up.sh --clc_cluster_name=k8s_3 --minion_type=standard --minion_count=10 --datacenter=VA1 -mem=6 -cpu=4
+```
+
+
+
+## Cluster Features and Architecture
+We configue the Kubernetes cluster with the following features:
+
+* KubeDNS: DNS resolution and service discovery
+* Heapster/InfluxDB: For metric collection. Needed for Grafana and auto-scaling.
+* Grafana: Kubernetes/Docker metric dashboard
+* KubeUI: Simple web interface to view kubernetes state
+* Kube Dashboard: New web interface to interact with your cluster
+
+We use the following to create the kubernetes cluster:
+
+* Kubernetes 1.1.7
+* Unbuntu 14.04
+* Flannel 0.5.4
+* Docker 1.9.1-0~trusty
+* Etcd 2.2.2
+
+## Optional add-ons
+
+* Logging: We offer an integrated centralized logging ELK platform so that all
+  kubernetes and docker logs get sent to the ELK stack. To install the ELK stack
+  and configure kubernetes to send logs to it, follow this documentation: [log
+  aggregation](log_aggregration.md). Note: We don't install this by default as
+  the footprint isn't trivial.
+
+## Cluster management
+
+The most widely used tool for managing a kubernetes cluster is the command-line
+utility _kubectl_.  If you do not already have a copy of this binary on your
+administrative machine, you may run the script _install_kubectl.sh_ which will
+download it and install it in _/usr/bin/local_.
+
+The script requires that the environment variable CLC_CLUSTER_NAME be defined
+
+_install_kubectl.sh_ also writes a configuration file which will embed the necessary
+authentication certificates for the particular cluster.  The configuration file is
+written to the  ${CLC_CLUSTER_HOME}/kube directory
+
+```
+export KUBECONFIG=${CLC_CLUSTER_HOME}/kube/config
+kubectl version
+kubectl cluster-info
+```
+
+### Accessing the cluster programmatically
+
+It's possible to use the locally-stored client certificates to access the api server
+```
+curl \
+   --cacert ${CLC_CLUSTER_HOME}/pki/ca.crt  \
+   --key ${CLC_CLUSTER_HOME}/pki/kubecfg.key \
+   --cert ${CLC_CLUSTER_HOME}/pki/kubecfg.crt  https://${MASTER_IP}:6443
+```
+But please note, this *does not* work out of the box with the curl binary
+distributed with OSX
```

Comment:
1. You don't have alt instructions?
2. End this sentence with a period.
3. wrap `curl` in backticks